CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 Changelog for Mantarray Waveform Analysis
 =========================================
 
+0.8.0 (2021-04-18)
+------------------
+
+- Added the ability to designate whether the data metrics in the pipeline template should be magnetic or force metrics
+
```

Comment:
Should the changelog include a note about fixing twitch direction? I noticed that's in the name of the PR but not mentioned here

Reply:
👍